packages/neuron-ui/src/widgets/Table/index.tsx

```diff
@@ -48,6 +68,7 @@ const Table = <T extends Record<string, any>>(props: TableProps<T>) => {
                   data-field={dataIndex}
                   align={align ?? 'left'}
                   className={headClassName}
+                  style={{ width, minWidth }}
```

Comment:
> How about using `className`?

I actually want to make these two properties a convenient generic property. So you don't have to use a specific class every time you use it. Because each column might have a different width, it would take a lot of classes to handle it, especially if I only had to set the width.

Reply:
Gotcha